app/assets/stylesheets/_variables.scss

```diff
@@ -19,4 +19,8 @@ $icon-star-stroke: #aaa !default;
 $icon-star-hover-fill: $icon-star-stroke !default;
 $icon-star-hover-stroke: $icon-star-stroke !default;
 $icon-star-active-fill: #f0ad4e !default;
-$icon-star-active-stroke: $icon-star-active-fill !default;
\ No newline at end of file
+$icon-star-active-stroke: $icon-star-active-fill !default;
+
+$octicon-pr-failure_error-color: #cb2431 !default;
```

Comment:
maybe we should reuse `$notification-open` and `$notification-closed` and add `$notification-pending` instead here?

Reply:
made the changes.